components/threatconnect.js

```diff
@@ -299,8 +317,10 @@ polarity.export = PolarityComponent.extend({
       let newCase = this.get(newCasePath) || {};
 
       const name = newCase.name ? newCase.name.trim() : '';
+      const description = newCase.desscription || '';
```

Comment:
@ahaias-tc Adding a description to the case isn't working due to this typo here.